src/Moat.hs

```diff
@@ -12,6 +12,7 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS_GHC -Wno-incomplete-record-updates #-}
```

Comment:
:thinking:

Reply:
Ahh, I guess this goes along with the incomplete pattern matching thing which is not actually incomplete.

Reply:
I don't think we want this.

Reply:
Will remove.